docs/docs/ias.md

```diff
@@ -1,41 +1,45 @@
-# Intel Attestation Service
+# Intel Attestation Service (EPID attestation only)
 
-!!! important
-    You don't need to read and perform these steps to deploy a Conclave app, unless you wish to use older
-    hardware that isn't capable of using the DCAP protocol. [Azure Gen 2 Virtual Machines](azure.md) in particular do not
-    need any of these steps.
+The Intel Attestation Service (IAS) helps you to verify that a remote computer isn't physically compromised and has the 
+latest security patches. IAS is used in the context of EPID attestation protocol. DCAP, the
+more recent attestation protocol, follows a different attestation flow
```

Comment:
Let's put a `.` at the end. Also it might be good to clarity `follows a different attestation flow (not discussed here below).`